repair/repair.cc

```diff
@@ -87,6 +87,15 @@ static std::vector<sstring> list_column_families(const database& db, const sstri
     return ret;
 }
 
+static std::vector<utils::UUID> get_table_ids(const database& db, const sstring& keyspace, const std::vector<sstring>& tables) {
+    std::vector<utils::UUID> table_ids;
+    table_ids.reserve(tables.size());
+    for (auto& table : tables) {
```

Comment:
@asias how many tables total can we expect here?
Every time I see a loop I think "stalls".

If this is running in context where we can futurize it (it seems we are), we should be futurizing this loop.

Reply:
It is depends on the number of tables in the keyspace. I agree it is better to futurize. I will do that when we clear other comments.